src/v2/view-builder/utils/ChallengeViewUtil.js

```diff
@@ -25,6 +25,8 @@ export function appendLoginHint(deviceChallengeUrl, loginHint) {
 export function doChallenge(view) {
   const deviceChallenge = view.getDeviceChallengePayload();
   const loginHint = view.options?.settings?.get('identifier');
+  const SPINNER_TIMEOUT = 4000;
+  const HIDE_CLASS = 'hide';
```

Comment:
since the name is not going to change for `hide`.. i would remove this unnecessary variable

Reply:
It's been used in multiple places, thus a variable.